src/every_query/train.py

```
@@ -58,19 +81,24 @@ def get_step(fp: Path) -> int:
     return sorted_checkpoints[-1] if sorted_checkpoints else None
 
 
-def collate_tasks(cfg: DictConfig) -> None:
+def collate_tasks(cfg: DictConfig) -> str:
     read_dir = f"{cfg.query.task_dir}/all"
-
-    task_str = f"{'|'.join(sorted(cfg.query.codes))}_{cfg.query.sample_times_per_subject}"
+ 
+    task_str = f"{'|'.join(sorted(cfg.query.codes))}_trainval{train_val_n}_heldout{held_out_n}"
```

Ruff (0.14.8)

Undefined name `train_val_n`

(F821)